model/database/registry.yaml

```
@@ -44,13 +50,18 @@ groups:
         brief: >
           The name of the operation or command being executed.
         note: >
-          It is RECOMMENDED to capture the value as provided by the application without attempting to do any case normalization.
+          It is RECOMMENDED to capture the value as provided by the application
+          without attempting to do any case normalization.
 
-          If the operation name is parsed from the query text, it SHOULD be the first operation name found in the query.
+          A single database query may involve multiple operations. If the operation
+          name is parsed from the query text, it SHOULD only be captured for queries that
+          contain a single operation or when the operation name describing the
+          whole query is available by other means (such as SQL comments).
```

it may be better not to specifically mention SQL comments yet since we will probably want to define a standard format(?)

```suggestion
          contain a single operation or when the operation name describing the
          whole query is available by other means.
```

---

I was also curious if any format or guidance on the sql comments would be added here